broker/consumer/ligo/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.pitt-broker</groupId>
+    <artifactId>lvk-stream</artifactId>
+    <version>0.1.0</version>
+    
+    <repositories>
+
+      <repository>
```

Comment:
Try adding the repository indicate here: 

https://github.com/jairsjunior/kafka-oauth?tab=readme-ov-file#kafka-client-configuration-producerconsumer